Formula/newrelic-cli.rb

```diff
@@ -0,0 +1,28 @@
+class NewrelicCli < Formula
+  desc "The New Relic Command-line Interface"
+  homepage "https://github.com/newrelic/newrelic-cli"
+  url "https://github.com/newrelic/newrelic-cli/archive/v0.2.3.tar.gz"
+  sha256 "f09d7e1705731578da79f0d9c204a6875edbd1d78cc6e69bd9f7d00dd9919986"
+  head "https://github.com/newrelic/newrelic-cli.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV.prepend_create_path "PATH", buildpath / "bin"
+    ENV["PROJECT_VER"] = version
+
+    dir = buildpath / "src/github.com/newrelic/newrelic-cli"
+    dir.install buildpath.children - [buildpath / ".brew_home"]
+
+    cd dir do
+      system "make", "compile-only"
+
+      bin.install "bin/darwin/newrelic"
+    end
+  end
+
+  test do
+    system "#{bin}/newrelic", "--help"
```

Comment:
It's better than nothing, but `--help` tests really should be avoided. See [here](https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula) for general guidance, including suggestions for cases where testing may be trickier.

Reply:
I've replaced this with a `newrelic config list` command.  Most of the commands here will require an API key to really validate.  But if we can list the users config (present or not), then that is at least exercising something.

Reply:
This has since been moved to `version` so that the output can be checked reliably.